charts/rate-limiter/values.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for pokt-rate-limiter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: pocketfoundation/rate-limiter
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "develop"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
```

Comment:
might be a nice-to-have to is to set a static role here, so that it makes configuring roles on ext secrets a bit simpler